libs/vm/src/sharded_state.cpp

```diff
@@ -71,7 +71,7 @@ class ShardedState : public IShardedState
   }
 
 private:
-  inline Ptr<String> ComposeFullKey(Ptr<String> const &key)
+  Ptr<String> ComposeFullKey(Ptr<String> const &key)
```

Comment:
That `inline` might be useful here (from performance perspective) to give compiler hint of our wish, since it this is _not_ templated class (= inlining is not implicit).

Reply:
Modern compilers do not need hints about what functions should be inlined. The `inline` keyword these days is only required as a directive for the linker to avoid apparent ODR violations.

Reply:
:+1: